lib/omniauth/strategies/twitch.rb

```diff
@@ -20,6 +20,12 @@ class Twitch < OmniAuth::Strategies::OAuth2
 
       option :authorize_options, [:scope]
 
+      def request_phase
+        redirect client.auth_code.
+          authorize_url({ redirect_uri: callback_url }.merge(authorize_params)).
```

Comment:
should `authorize_params` be `authorize_options`??

Reply:
okay but does the refactor still do what you expect?

Reply:
Yes it does. It allows for the use of additional scopes, as in here: https://github.com/rigel-eva/CmdrShepard/blob/5fa4b5ad2aa31a31a80b4d2c7df2b36d89f4fc56/webserver/config/initializers/omniauth.rb#L7